packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -46,10 +46,6 @@ BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.4.1
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 9.0.0
-BuildRequires: %{?scl_prefix}npm(graphql-tag) >= 2.11.0
-BuildRequires: %{?scl_prefix}npm(graphql-tag) < 3.0.0
-BuildRequires: %{?scl_prefix}npm(graphql) >= 15.5.0
-BuildRequires: %{?scl_prefix}npm(graphql) < 16.0.0
```

Comment:
Do these buildrequires need to be dropped?  Looks like the rpm failed to build due to missing graphql-tag:

```
ModuleNotFoundError: Module not found: Error: Can't resolve 'graphql-tag/loader' in '/builddir/build/BUILD/foreman_ansible-7.0.4/usr/share/foreman'
```

Reply:
Note to myself: Do not blindly trust bump_rpm.sh does the right thing

Reply:
Haha, wise words.